### DISCUSSION TITLE
ENTSO-E price presentation - possibly wrong in 2.2.23

### DISCUSSION THREAD

**Discussion by torkelmj** [UtilitechAS/amsreader-firmware#669] [Q&A]
Howdy.

I just noticed a discrepancy between the price information for region NO1, as reported by my supplier of electrical energy and as presented by the AMS reader (Pow-U) running FW 2.2.23. Never noticed any discrepancies when running FW v. 2.2.21. 

Attempted a reboot of the device, still the same result. 

Price per kWh as reported in the AMS reader UI:
![AMS_Price](https://github.com/UtilitechAS/amsreader-firmware/assets/43777466/7a8ca0c5-982c-4e6d-a351-b2166508c089)

Confirming correct region set for the ENTSO-E API: 
![AMS_ENTSO](https://github.com/UtilitechAS/amsreader-firmware/assets/43777466/a993d135-d540-4de0-8f4e-a55f40cdddb4)

Price per kWh as reported by Fortum (and others): 
![AMS_Fortum](https://github.com/UtilitechAS/amsreader-firmware/assets/43777466/c640c88b-22f2-4e60-bffb-cab5052ddd16)

I have no indication whether the data returned by the ENTSO-E API is wrong or whether the presentation by AMS reader is faulty. But the price information simply doesn't match.

Is there a time stamp in the data returned by the ENTSO-E API, which could be used to verify that the supplied price information is actually current?

Anyone else?

**Comment by torkelmj** [comment]
Could also be an idea to change the "Future energy price (øre)" caption into "Future energy price (øre) per kWh", just for clarity ?

**Comment by ArnieO** [comment]
Same problem as this issue maybe? https://github.com/UtilitechAS/amsreader-firmware/discussions/669
There is a test fix in that thread, it would be great if you could try it.

**Comment by torkelmj** [comment]
The problem self-corrected a couple of hours after the reboot. 

Still, a suggestion for the caption text - I think it could be useful to know that the presented prices are actually current/reasonably updated:
![AMS_Price](https://github.com/UtilitechAS/amsreader-firmware/assets/43777466/4a39ceb9-8fdc-4078-83d3-88cdfe7ac585)

Thanks for the mighty fine work you're doing.

**Reply by ArnieO** [reply]
Thank you for kind words and your suggestion to improve the caption text!
